caffe2/core/macros.h

```diff
@@ -1,4 +0,0 @@
-#pragma once
```

Comment:
Please exclude this file from the PR

Reply:
Hum ... I did remove this file in 3d9bdd0